api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractService.java

```diff
@@ -35,11 +35,10 @@
 
 
     /**
-     * Loop through all donations for a contract and either void or capture those payments
+     * Changes the state of a contract.
      *
      * @param contractModel contract to settle payment for
-     * @param shouldReleasePayments flag indicating if payments should be released to the streamer. If false the payment
-     *                              Is voided and never leaves the donator's account.
+     * @param newContractState The new contract state after the result of a vote.
      */
-    void settlePayments(ContractModel contractModel, boolean shouldReleasePayments);
+    void setContractState(ContractModel contractModel, ContractState newContractState);
```

Comment:
Would we ever set state without this method? like we change the models state directly and then save using the repo?

Reply:
Talked offline. Lets clarify that all state transitions are expected to use this method. additionally maybe try and deprecate the model setter